quiz.js

```diff
@@ -47,24 +47,29 @@ var characterField = document.getElementById("character");
 characterField.addEventListener("keypress", handleKeyPress);
 
 
+
 function handleKeyPress(e){	 // when you press enter instead of click, had to add the function name to both the input tags
- var key=e.keyCode;
-  if (key==13){
-  	movement();
+  var key=e.keyCode;
+  	if (key==13){
+  	  movement();
   }
 }
 
+var treeData = {}; // needs to be globally defined not related to the function movement
```

Comment:
In this case, it does _not_ need to be globally defined, since, even though you're using it in the ```tree``` function as well, you're directly _passing it in_ to the ```tree``` function as an argument on line 63